docs/visualization/viz_module/blender/Getting started.rst

```diff
@@ -0,0 +1,129 @@
+Getting started
+----------------
+
+Installation
+^^^^^^^^^^^^
+
+Blender offers the possibility to control things using python. However, it has its own python interpreter and it's not easy to
+install new packages in it. If we want to use ``sisl`` within it, our best bet is to create a virtual environment and then let
+blender use that python interpreter. The environment that we create, though, **MUST have exactly the same python version** as
+the one shipped with blender.
+
+Following, you have a step by step guide to get blender ready for plotting with sisl:
+
+1. **Install blender**. You can install by downloading it directly from their official webpage, or in any other way. 
+Check `their installation documentation <https://docs.blender.org/manual/en/latest/getting_started/installing/index.html>`_
+
+In ubuntu we can install it with::
+
+    snap install blender    
+
+2. **Find out blender's python version**. You should check what is the version that blender is
+shipped with. Being `blender` the name of the executable, you can run::
+       
+    blender -b --python-expr "import sys; print(f'PYTHON VERSION: {sys.version}')"
+
+In blender 2.93 it gives an output that looks like this::
+
+    Blender 2.93.4 (hash b7205031cec4 built 2021-08-31 23:36:18)
+    Read prefs: /home/.config/blender/2.93/config/userpref.blend
+    PYTHON VERSION: 3.9.2 (default, Feb 25 2021, 12:19:39) 
+    [GCC 9.3.1 20200408 (Red Hat 9.3.1-2)]
+
+    Blender quit
+
+Therefore, we know that **blender 2.93.4 uses python 3.9.2.**
+
+3. **Create an environment with that python version** and install sisl (*skip if you have it already*).
+In this case, we will use conda as the environment manager, since it lets us very easily select the python version.
+You probably don't need the exact micro version. In our case asking for ``3.9`` is enough::
+    
+    conda create -n blender-python python=3.9
+
+Then install all the packages you want to use in blender::
+
+    conda activate blender-python
+    pip install sisl[viz]
```

Comment:
the pip community is preferring `python -m pip install` because this ensures the pip module that is used corresponds to the python running. I'll merge and update.

Reply:
True, sorry!